packages/serverpod/lib/src/server/server.dart

```diff
@@ -8,6 +8,7 @@ import 'package:serverpod/src/database/database.dart';
 import 'package:serverpod/src/database/database_pool_manager.dart';
 import 'package:serverpod/src/server/health_check.dart';
 import 'package:serverpod/src/server/websocket_request_handlers/endpoint_websocket_request_handler.dart';
+import 'package:serverpod/src/server/websocket_request_handlers/method_websocket_request_handler.dart';
 
 import '../cache/caches.dart';
```

Comment:
Nit: change to package import. Don't we have linting rules for this?

Reply:
No, I don't think we have a rule for this so far. We kind of patch these up as we go. But will fix! 👍